### PR TITLE
drop nix support

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@hswong3i @rhatdan @vrothberg @mtrmac @zhangguanzhang PTAL

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Review by vrothberg — COMMENTED** [review]
Why? Please add a description/explanation to the commit.

**Comment by lsm5** [comment]
> Why? Please add a description/explanation to the commit.

I do have a `nix build is no longer being maintained`. Does that suffice or do you need something more?

**Comment by lsm5** [comment]
@vrothberg also, do you know whatsup with the skopeo test fail?

**Comment by vrothberg** [comment]
> > Why? Please add a description/explanation to the commit.
> 
> I do have a `nix build is no longer being maintained`. Does that suffice or do you need something more?

Apologies, my bad.  Yes that works, thank you!

**Comment by vrothberg** [comment]
> @vrothberg also, do you know whatsup with the skopeo test fail?

Looks like a registry flake to me. I will restart

**Review by vrothberg — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -132,22 +131,6 @@ help:
...
-	@nix run \
-		-f channel:nixos-21.05 nix-prefetch-git \
-		-c nix-prefetch-git \
-		--no-deepClone \
-		https://github.com/nixos/nixpkgs refs/heads/nixos-21.05 > nix/nixpkgs.json
-
-# Build statically linked binary
-.PHONY: static
-static:
-	@nix build -f nix/
```
Comment (mtrmac) [inline Makefile:147]: ACK to the removal; shouldn’t the whole `nix` subdirectory be removed as well?

**Pull request merged**